cmd/mount.go

```diff
@@ -110,9 +110,22 @@ func (opt *mountOptions) validate(skipEmptyMount bool) error {
 	if err := common.ELogLevel.Parse(opt.Logging.LogLevel); err != nil {
 		return fmt.Errorf("invalid log level [%s]", err.Error())
 	}
-	opt.Logging.LogFilePath = os.ExpandEnv(opt.Logging.LogFilePath)
+
+	if opt.DefaultWorkingDir != "" {
+		common.DefaultWorkDir = opt.DefaultWorkingDir
+
+		if opt.Logging.LogFilePath == common.DefaultLogFilePath {
+			// If default-workng-dir is set then default log path shall be set to that path
+			// Ignore if specific log-path is provided by user
```

Comment:
in our readme we should mention that this param overwrites the default working dir when it comes to logging

Reply:
That was the expectation earlier as well, it's just that it was not honored in GA release.